src/main/java/com/jcabi/github/mock/MkIssueLabels.java

```diff
@@ -182,4 +217,15 @@ private String xpath() {
         );
     }
 
+    /**
+     * Returns a set of all of the issue's labels.
+     * @return Set of label names
+     */
+    private Set<String> labelSet() {
```

Comment:
@carlosmiranda 

> And let's name this method `labels()`

Done.

> As far as I can tell, there's no particular reason why it should declare the return type `Set`

Logically speaking, it is a set; it cannot have duplicates; multiple labels, by definition, cannot share the same name.